admin/tabs/general/roles/details/members/RoleMembers.ts

```diff
@@ -23,36 +23,40 @@ import {isEmpty, sum, values} from 'lodash';
 import {ReactNode} from 'react';
 
 export interface RoleMembersProps extends HoistProps<RoleMembersModel> {
-    showEffective: boolean;
+    showInherited: boolean;
 }
 
 export const roleMembers = hoistCmp.factory<RoleMembersProps>({
     className: 'xh-admin-role-members',
     displayName: 'RoleMembers',
     model: creates(RoleMembersModel),
-    render({className, model, showEffective}) {
-        const {directCounts, effectiveCounts} = model;
+    render({className, model, showInherited}) {
+        const {inheritedRolesCount, effectiveCounts} = model;
         return panel({
             className,
             tbar: [
                 buttonGroupInput({
                     bind: 'activeTabId',
+                    outlined: true,
                     items: [
                         button({
-                            text: buttonText({text: 'Assigned', countsByType: directCounts}),
-                            value: 'directMembers'
+                            text: buttonText({text: 'Members', countsByType: effectiveCounts}),
+                            value: 'effectiveMembers'
                         }),
                         button({
-                            text: buttonText({text: 'Effective', countsByType: effectiveCounts}),
-                            value: 'effectiveMembers'
+                            text: buttonText({
```

Comment:
The `buttonText` component renders the word `Members` in its tooltip.  That made sense for direct / vs effective members, but is not really accurate for `inheritedRoles` (inherited roles are not members).

Could update the `buttonText` component, but since we only have a single count (number of inherited roles), I'm not sure it actually makes sense to re-use it here.